stock_critical_forecast/tests/TEST_INSTRUCTIONS.rst

```diff
@@ -0,0 +1,6 @@
+- Open the Inventory app
+- Under Operations menu click "Calculate Critical Forecast"
+- Under Reporting menu open the Critical Forecast
+- Critical forecast is shown for products in inventory, manufacturing and purchase
```

Comment:
This is not correct.

"The critical forecast by default shows products with a critical order date"